tests/pytest/conftest.py

```diff
@@ -1,7 +1,34 @@
+from django.contrib.sessions.middleware import SessionMiddleware
+
 import pytest
 
 
 @pytest.fixture(scope="session")
 def django_db_setup():
     # use existing database since it's read-only
     pass
+
+
+@pytest.fixture
+def session_request(request, rf):
+    """
+    Fixture creates a new Django request object and initializes its session.
+
+    Optionally use @pytest.mark.request_path(path: str) to customize the request's path.
+    """
+
+    # see if a request_path was provided via marker
+    marker = request.node.get_closest_marker("request_path")
```

Comment:
Hmm, having to specify the path outside of the test is strange to me. I could also imagine test cases where we'd need to simulate multiple requests. I personally prefer the helper function approach 🤷

Reply:
Can you say more about your hesitation here?

The path is specified next to / on top of the test definition...

```python
@pytest.mark.request_path("/some/path")
def test_something(session_request):
    # now session_request is initialized for /some/path and with a session

```

In many cases (like, all of these tests plus everything in #474 so far) the path actually doesn't matter? We just need a request with a session initialized.

Doing this as a fixture means it can easily be injected into any test and is already ready to go, no extra set up needed in the test itself.

Reply:
How about both!? 

See [`384b332`](https://github.com/cal-itp/benefits/pull/436/commits/384b332f0feda3d066e5f508dbe02e3f34eda370)